FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -7,31 +7,45 @@
  of patent rights can be found in the PATENTS file in the same directory.
  */
 
+#import "FBKVOController.h"
 #import "FBTweakCollection.h"
 #import "FBTweakCategory.h"
 #import "FBTweak.h"
+#import "_FBKeyboardManager.h"
+#import "_FBTweakArrayViewController.h"
 #import "_FBTweakCollectionViewController.h"
-#import "_FBTweakTableViewCell.h"
 #import "_FBTweakColorViewController.h"
 #import "_FBTweakDictionaryViewController.h"
-#import "_FBTweakArrayViewController.h"
-#import "_FBKeyboardManager.h"
-
-@interface _FBTweakCollectionViewController () <UITableViewDelegate, UITableViewDataSource>
-@end
+#import "_FBTweakTableViewCell.h"
 
 @implementation _FBTweakCollectionViewController {
-  UITableView *_tableView;
-  NSArray *_sortedCollections;
+   NSArray<FBTweakCollection *> *_sortedCollections;
   _FBKeyboardManager *_keyboardManager;
+  FBKVOController *_kvoController;
 }
 
-- (instancetype)initWithTweakCategory:(FBTweakCategory *)category
+- (instancetype)initWithTweakCategory:(id<FBTweakCategory>)category
 {
   if ((self = [super init])) {
     _tweakCategory = category;
+    self.clearsSelectionOnViewWillAppear = YES;
     self.title = _tweakCategory.name;
-    [self _reloadData];
+
+    _kvoController = [FBKVOController controllerWithObserver:self];
+    [_kvoController observe:_tweakCategory keyPath:@"tweakCollections"
+                        options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
+                          block:^(id observer __unused, id object  __unused,
```

Comment:
remove the non necessary `__unused` along with argument names.

Reply:
can't do - the same issue about cpp.